source/slang/check.cpp

```diff
@@ -2604,9 +2604,16 @@ namespace Slang
                 else if (auto bindingAttr = as<GLSLBindingAttribute>(attr))
                 {
                     SLANG_ASSERT(attr->args.Count() == 2);
+
                     auto binding = checkConstantIntVal(attr->args[0]);
                     auto set = checkConstantIntVal(attr->args[1]);
 
+                    if (!binding || !set)
+                    {
+                        getSink()->diagnose(attr, Diagnostics::attributeExpectsTwoIntArgs, attr->name);
```

Comment:
Hmm... Shouldn't the assert and the two `checkConstantIntVal` calls above have verified all of this?